spec/index.bs

```diff
@@ -16512,6 +16549,22 @@ This feature adds the following [=optional API surfaces=]:
 - New WGSL extensions:
     - [=extension/dual_source_blending=]
 
+<h3 id=subgroups data-dfn-type=enum-value data-dfn-for=GPUFeatureName>`"subgroups"`
+<span id=dom-gpufeaturename-subgroups></span>
+</h3>
+
+Allows the use of the subgroup and quad operations in WGSL.
+
+This feature adds the following [=optional API surfaces=]:
+- New entries in {{GPUAdapterProperties}}
+    - {{GPUAdapterProperties/minSubgroupSize}}
+    - {{GPUAdapterProperties/maxSubgroupSize}}
+
+- New WGSL extensions:
+    - [=enable-extension|subgroups=]
+
+Issue: replace extension link when spec lands.
```

Comment:
I assume this means replace `[=enable-extension|subgroups=]` with `[=subgroups=]`? That can be done in the same PR since the definition is "fake" anyway (it gets it from the `<pre class=anchors>`, doesn't hit the specref database)